app.js

```diff
@@ -1,7 +1,77 @@
-import Config from "./config";
-import {User, Product} from "./models";
+const express = require('express');
+const uniqid = require('uniqid');
+const bodyParser = require('body-parser');
 
-const user = new User();
-const product = new Product();
+const app = express();
 
-console.log(Config.name)
\ No newline at end of file
+app.use(cookieParser);
+app.use(queryParser);
+app.use(bodyParser.json());
+
+const products = [];
+const users = [];
+
+app.get('/api/products', (req, res) => {
+    res.json(JSON.stringify(products));
+});
+  
+app.get('/api/products/:id', (req, res) => {
+    for (let i = 0; i < products.length; i++) {
+        if (products[i].id === req.params.id) {
+            res.json(JSON.stringify(products[i]));
+            return;
+        }
+    }
+    res.send(`product ${req.params.id} doesnt exist`);
+});
+
+app.get('/api/products/:id/reviews', (req, res) => {
+    for (let i = 0; i < products.length; i++) {
+        if (products[i].id === req.params.id) {
+            res.json(JSON.stringify(products[i].reviews));
+            return;
+        }
+    }
+    res.send(`product ${req.params.id} doesnt exist`);
+});
+  
+app.post('/api/products', (req, res) => {
+    console.log(req.body)
+    let product = {
+        ...req.body,
+        id: uniqid(),
+    }
+    products.push(product)
+    res.json(JSON.stringify(product));
+})
+
+app.get('/api/users', (req, res) => {
+    res.json(JSON.stringify(users));
+})
+
+function cookieParser(req, res, next) {
```

Comment:
Please move this function in the `middlewares` folder as in the project structure you should have it